warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IngestTypeVisitor.java

```diff
@@ -262,7 +260,7 @@ public Set<String> getIngestTypesForJunction(JexlNode node) {
     public Set<String> getIngestTypesForLeaf(JexlNode node) {
```

Comment:
Why did we create a separate visitor for this?  Why not use the visitor that pulls fields from the query (which we duplicate for the most part in here), and then simply get all of the datatypes for that list of fields?  I feel this may be fragile as there may be changes to the query fields visitor that would not get incorporated in here and it would be a long time before that would be discovered.

Reply:
Updated the docs on the visitor per our conversation